src/langs/en.json

```
@@ -173,27 +180,46 @@
   "LIBRARY_SETTINGS_UNPUBLISH_BUTTON": "Unpublish",
-  "LIBRARY_SETTINGS_VALIDATION_CONFIGURATION_INFORMATIONS": "Set the options for your resource to make it easily accessible to the public.",
   "LIBRARY_SETTINGS_VALIDATION_CONFIGURATION_TITLE": "Item attributes",
   "LIBRARY_SETTINGS_VALIDATION_INFORMATIONS": "You need to validate your item before publishing it. Please allow some time for the validation to finish.",
-  "LIBRARY_SETTINGS_VALIDATION_PUBLICATION_INFORMATIONS": "Once your item is validated, you can publish your item by clicking the button.",
   "LIBRARY_SETTINGS_VALIDATION_PUBLICATION_TITLE": "Publication",
   "LIBRARY_SETTINGS_VALIDATION_REFRESH_BUTTON": "Refresh",
-  "LIBRARY_SETTINGS_VALIDATION_STATUS_FAILURE": "Your item might contain inappropriate content. Please remove them and validate your item again. If you have any question, please contact {{contact, string}}.",
+  "LIBRARY_SETTINGS_VALIDATION_STATUS_FAILURE": "Your item might contain inappropriate content. Please remove them and validate your item again. If you have any question, please contact {{contact}}.",
   "LIBRARY_SETTINGS_VALIDATION_STATUS_PENDING_AUTOMATIC": "Your item is pending automatic validation. Once the validation succeeds, you will be able to publish your item.",
-  "LIBRARY_SETTINGS_VALIDATION_STATUS_PENDING_MANUAL": "Your item failed the automatic validation. A member of our team will manually check your collection, please wait for the result.",
+  "LIBRARY_SETTINGS_VALIDATION_STATUS_READY_TO_PUBLISH": "The item is valid and ready to be published in the Library.",
+  "LIBRARY_SETTINGS_VALIDATION_STATUS_OUTDATED": "The element was updated since the last verification. It must be validated again to be sure that it is still valid.",
   "LIBRARY_SETTINGS_VALIDATION_TITLE": "Validation",
   "LIBRARY_SETTINGS_VALIDATION_VALIDATE_BUTTON": "Validate",
-  "LIBRARY_SETTINGS_VISIBILITY_INFORMATIONS": "This element must be public to be published",
+  "LIBRARY_SETTINGS_VISIBILITY_INFORMATIONS": "The published element is not public anymore. The item have to be public to be available in the Library.",
+  "LIBRARY_SETTINGS_VISIBILITY_CHANGE_BUTTON": "Change the visibility",
+  "LIBRARY_SETTINGS_RETRY_BUTTON": "Retry",
+  "LIBRARY_SETTINGS_VIEW_LIBRARY_BUTTON": "View on Library",
+  "LIBRARY_SETTINGS_UNAUTHORIZED": "You do not have the permission to publish this item.",
+  "LIBRARY_SETTINGS_ITEM_DESCRIPTION_LABEL": "Item's description",
+  "LIBRARY_SETTINGS_ITEM_DESCRIPTION_MISSING_WARNING": "Add a description to explain what your item is about",
+  "LIBRARY_SETTINGS_THUMBNAIL_MISSING_WARNING": "Add a thumbnail to improve your publication's visibility",
+  "LIBRARY_SETTINGS_PUBLICATION_STATUS_LOADING": "Loading status...",
+  "LIBRARY_SETTINGS_PUBLICATION_STATUS_PUBLISHED": "Published",
+  "LIBRARY_SETTINGS_PUBLICATION_STATUS_PUBLISHED_CHILDREN": "Published by a Parent",
```

```suggestion
  "LIBRARY_SETTINGS_PUBLICATION_STATUS_PUBLISHED_CHILDREN": "Published from above",
```
not best suggestion

---

```suggestion
  "LIBRARY_SETTINGS_PUBLICATION_STATUS_PUBLISHED_CHILDREN": "Part of a published collection",
```

@pyphilia what do you think ?